doc/sphinx/source/recipes/recipe_combined_indices.rst

```diff
@@ -0,0 +1,76 @@
+.. _recipes_combined_indices:
+
+Indices based on area averages
+====================================================
+
+Overview
+--------
+
+The goal of this diagnostic is to compute indices based on area averages.
+
+In recipe_combined_indices.yml, after defining the period (historical or
+future projection), the variable is selected. The predefined indices are:
+
+* Nino 3
+* Nino 3.4
+* Nino 4
+* North Atlantic Oscillation (NAO)
+* Southern Oscillation Index (SOI)
+
+Available recipes and diagnostics
+-----------------------------------
+
+Recipes are stored in recipes/
+
+* recipe_combined_indices.yml
+
+Diagnostics are stored in diag_scripts/magic_bsc/
+
+* combined_indices.r : calculates the area-weighted means and multi-model means, with or without weights
+
+
+
+User settings
+-------------
+
+User setting files are stored in recipes/
+
+#. recipe_combined_indices.yml
+
+   *Required settings for script*
+
+   * region: one of the following strings Nino3, Nino3.4, Nino4, NAO, SOI
+   * running_mean: an integer specifying the length of the window (in months) to be used for computing the running mean.
```

Comment:
It's not clear how to switch between running mean and seasonal mean based on moninf/monsup. Could you clarify that?